plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoClientConfig.java

```diff
@@ -237,4 +239,17 @@ public MongoClientConfig setMaxConnectionIdleTime(int maxConnectionIdleTime)
         this.maxConnectionIdleTime = maxConnectionIdleTime;
         return this;
     }
+
+    public boolean isProjectionPushdownEnabled()
+    {
+        return projectionPushDownEnabled;
+    }
+
+    @Config("mongodb.projection-pushdown-enabled")
+    @ConfigDescription("Read only required fields from a struct")
```

Comment:
No, It should be actually `row`. But since this config property has `struct` in the description for all other connectors so I opt for `struct`. I will add a pre-commit to change `struct` to `row`.